src/ui/button.ts

```diff
@@ -102,13 +105,15 @@ export class MenuButton {
 
     engine.addSystem(() => {
       // TODO: Why is not been triggered ?
-      if (tweenSystem.tweenCompleted(this.button)) {
-        console.log('asd', Tween.getOrNull(this.button))
-      }
+      // if (tweenSystem.tweenCompleted(this.button)) {
+      //   console.log('asd', Tween.getOrNull(this.button))
+      // }
 
       // TODO: this should be tweenCompleted but no idea why is not working :sadcat:
-      if (TweenState.getOrNull(this.button)?.currentTime === 1 && TweenSequence.getOrNull(this.button)) {
+      // if (TweenState.getOrNull(this.button)?.currentTime === 1 && TweenSequence.getOrNull(this.button)) {
+      if (tweenSystem.tweenCompleted(this.button)) {
```

Comment:
it works ? Maybe because i had the lib linked, but good to know.

you can remove the comments if you want

Reply:
it works! actually i was having the "asd" message in console since many weeks ago.